### ISSUE TITLE
Change settings on contact page not possible: Could not locate selected profile.

### ISSUE THREAD

**Issue by efelon** [friendica/friendica#6610]
### Expected behavior

Changing "Contact Settings" of a single contact (e.g. Hide this contact from others) on it's local page (e.g. https://myfriendicaserver.de/contact/8) should just work.

### Actual behavior

I always get this error:

![image](https://user-images.githubusercontent.com/10722552/52490131-5586e900-2bc4-11e9-9e02-38a1348d530c.png)

Only related line from the logs:

```
2019-02-07T20:36:40Z@*******************	[Info]:DBA.php:555:p	DB Error 1054: Unknown column 'deleted' in 'where clause'
DBA::exists, Contact::post, call_user_func, App::runFrontend
SELECT `id` FROM `profile` WHERE (`id` = 2 AND `uid` = 2 AND `deleted` = 0) LIMIT 1
```

### Steps to reproduce the problem

This is always true for every user on this server (admin and normal user). Select any of the users contacts to change any setting displayed above and click "submit". I also checked different themes.

### Workaround

This is the HTML part of the profile selection on the page shown on the screenshot:

```
<select id="contact-profile-selector" class="form-control" name="profile-assign">
<option value="1" selected="selected">Standard</option>
</select>
```

If i change value="1" to value="Standard" I can submit the changes successfully.

### Friendica version you encountered the problem

Friendica, version 2019.01 
Database version is 1293, the post update version is 1281.

Installed addons/apps:
buglink, calc, communityhome, curweather, diaspora, highlightjs, leistungsschutzrecht, morepokes, openstreetmap, pumpio, showmore, smileybutton, statusnet, unicode_smilies

### Friendica source (git, zip)

zip

### PHP/SQL ... versions

Web Server | nginx/1.10.3
PHP | 7.0.33-0+deb9u1
MySQL | 10.1.37-MariaDB-0+deb9u1

**Comment by MrPetovan** [comment]
Thanks for the detailed report!

**Comment by rabuzarus** [comment]
Thanks for your report but it should be already fixed in the `develop` branch and will be included in the next stable realease

**Comment by efelon** [comment]
@MrPetovan You're welcome. 
And @rabuzarus sorry, I have not thought about searching also the PRs.

**Comment by MrPetovan** [comment]
Fixed by #6484
